configure.ac

```diff
@@ -132,11 +132,17 @@ if test -z $GCAB ; then
 fi
 
 # ColorHug support
+PKG_CHECK_MODULES(COLORHUG, colorhug >= 1.2.12,
+		  has_colorhug=yes,
+		  has_colorhug=no)
 AC_ARG_ENABLE(colorhug, AS_HELP_STRING([--enable-colorhug],[Enable ColorHug support]),
 	      enable_colorhug=$enableval, enable_colorhug=yes)
```

Comment:
So, I thought the way we were supposed to do this with automake was make

    enable_colorhug=auto)

then:

    if enable_colorhug==yes and not have_colorhug -> error
    if enable_colorhug==no -> no ifdefs
    if enable_colorhug==yes and have_colorhug -> define ifdefs
    if enable_colorhug==auto and have_colorhug -> define ifdefs
    if enable_colorhug==auto and not have_colorhug -> no ifdefs
    
This lets people have the autodetection by default, and also allows them to force-off (--disable-colorhug) and force-on (--enable-colorhug) for distro spec files for example, where we want to be notified about missing deps.

I think the configure.ac has this in gnome-software if you want to copy it.